L1Trigger/L1THGCal/interface/be_algorithms/HGCalClusteringImpl.h

```diff
@@ -14,11 +14,11 @@ class HGCalClusteringImpl{
   
     HGCalClusteringImpl( const edm::ParameterSet & conf);    
 
-    void clusterize( const edm::OrphanHandle<l1t::HGCalTriggerCellBxCollection> triggerCellsHandle,
+    void clusterize( const edm::PtrVector<l1t::HGCalTriggerCell> triggerCellsPtrs,
                      l1t::HGCalClusterBxCollection & clusters 
         );
 
-    bool isPertinent( const l1t::HGCalTriggerCell &tc, l1t::HGCalCluster &clu, double distXY) const;
+    bool isPertinent( const edm::Ptr<l1t::HGCalTriggerCell> tc, const l1t::HGCalCluster &clu, double distXY) const;
```

Comment:
The first parameter was fine before (`const  l1t::HGCalTriggerCell &tc`) . It is not needed to have an edm::Ptr here, so standard arguments should be preferred.